identity-verification/java/src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+spring.application.name=Identity Verification Sample
+falu.apiKey=
```

Comment:
Would a placeholder value here still work? e.g. `falu.apiKey="<your-secret-api-key-here>"`

Reply:
Most popular thing is to leave it blank.